src/pages/AppDetails/AppInfo/AppDescription.tsx

```diff
@@ -126,26 +131,33 @@ class AppDescription extends React.Component<AppDescriptionProps, AppDescription
         title={app.name}
         istio={this.istioSidecar()}
         items={
-          <Row>
-            <Col xs={12} sm={6} md={4} lg={4}>
-              <ListView>{this.workloadList()}</ListView>
-            </Col>
-            <Col xs={12} sm={6} md={4} lg={4}>
-              <ListView>{this.serviceList()}</ListView>
-            </Col>
-            <Col xs={0} sm={0} md={1} lg={1} />
-            <Col xs={12} sm={6} md={3} lg={3}>
-              <div className="progress-description">
-                <strong>Health</strong>
-              </div>
-              <HealthIndicator
-                id={app.name}
-                health={this.props.health}
-                mode={DisplayMode.LARGE}
-                tooltipPlacement="left"
-              />
-            </Col>
-          </Row>
+          <>
+            {app.name && (
+              <Row>
+                <Link to={this.showOnGraphLink(app.name)}>Show on graph</Link>
```

Comment:
Probably, the `<Col>` may fix i ndentation/margin.

Reply:
Most likely, will try that

Reply:
Fix this